src/store/modules/news.store.js

```diff
@@ -0,0 +1,20 @@
+import categoryStateInit from "./state.init";
+
+export default {
+	namespaced: true,
+	state: {
+		category: categoryStateInit("aktualnosci", 2, 1),
+		payload: []
+	},
+	mutations: {
+		loadPayload(state, payload) {
+			state.payload = payload;
+		}
+	},
+	actions: {
+		async getPaginationPayload(context) {
```

Comment:
the name suggests that the only kind of data that we can retrieve here is the payload number itself. Is this really the case here?

Reply:
Really ? For me it is payload in pagination mode